test/.jshintrc

```diff
@@ -3,6 +3,7 @@
 	"qunit": true,
 
 	"-W053": true,
+	"-W097": true, // disable warning for 'Use the function form of "use strict".'
```

Comment:
According to https://github.com/jshint/jshint/issues/741 , the file can contain comments. Should I still remove it?